Makefile

```diff
@@ -497,7 +497,8 @@ fpga: $(ariane_pkg) $(util) $(src) $(fpga_src) $(uart_src)
 	@echo read_verilog -sv {$(filter-out $(fpga_filter), $(src))} 	   >> fpga/scripts/add_sources.tcl
 	@echo read_verilog -sv {$(fpga_src)}   >> fpga/scripts/add_sources.tcl
 	@echo "[FPGA] Generate Bitstream"
-	cd fpga && make BOARD="genesys2" XILINX_PART="xc7k325tffg900-2" XILINX_BOARD="digilentinc.com:genesys2:part0:1.1" CLK_PERIOD_NS="20"
+	cd fpga && make BOARD="kc705" XILINX_PART="xc7k325tffg900-2" XILINX_BOARD="xilinx.com:kc705:part0:1.5" CLK_PERIOD_NS="20"
```

Comment:
We probably need some more graceful handling here. Maybe we can pass a `board` parameter on which we select the appropriate part and board.

Reply:
Yeah, I agree (un-/)commenting the respective lines isn't pretty. I couldn't figure out the way you'd want this to be handled by looking at the code for the VCU118.
An export with the board name, maybe defaulting to the genesys2, seems like a good approach though.

Reply:
kc705 and GenesysII share the same Kintex FPGA, but not the same Ethernet. So perhaps the Ethernet needs to be packaged differently. I suggest a toplevel which instances Ethernet and SRAM interface, along with a fairly generic Ariane top level which mops up all the unix platform stuff like the PLIC and DM_TOP. You also need to make build directories depend on the architecture to prevent confusing the Vivado tool.

Reply:
> We probably need some more graceful handling here. Maybe we can pass a `board` parameter on which we select the appropriate part and board.

I updated the Makefile to use the BOARD environmental variable to set the `BOARD`, `XILINX_PART`, `XILINX_BOARD` and `CLK_PERIOD_NS` parameters. It still defaults to the genesys2 board.